### PR TITLE
Add 1ES pipeline for testing.

### PR DESCRIPTION
**Pull request by philliphoff**

### CODE REVIEW

File: .azure-pipelines/common/setup.yml
Comment (bwateratmsft) [inline .azure-pipelines/common/setup.yml:1]: This file is no longer used. Instead, you'll need to create a `.nvmrc` file at the repo root.

Comment (philliphoff) [reply .azure-pipelines/common/setup.yml:1]: I plan to refactor the former build system once the 1ES build system is in place and running, likely borrowing from the other extensions.

Comment (bwateratmsft) [reply .azure-pipelines/common/setup.yml:1]: Ok. You'll still want an .nvmrc file.

Comment (philliphoff) [reply .azure-pipelines/common/setup.yml:1]: Added.

**Review by bwateratmsft — APPROVED** [review]

**Pull request merged**